public/app/plugins/datasource/loki/types.ts

```diff
@@ -2,6 +2,9 @@ import { DataQuery, Labels, DataSourceJsonData } from '@grafana/ui/src/types';
 
 export interface LokiQuery extends DataQuery {
   expr: string;
+  live?: boolean;
```

Comment:
Should these props be here or in LokiOptions?

Reply:
I think it makes sense as a query option, not as a DataSource option.  We should only stream results when we ask to stream them